tests/test_mpirun.py

```diff
@@ -10,8 +10,9 @@
 def test_mpi_run():
     if MPI:
         repeatdt = 200*86400
-        os.system('mpirun -np 2 python parcels/examples/example_stommel.py -p 4 -o StommelMPI.nc -r %d' % repeatdt)
-        os.system('python parcels/examples/example_stommel.py -p 4 -o StommelNoMPI.nc -r %d' % repeatdt)
+        parcels_path = os.path.dirname(__file__) + '/../parcels'
```

Comment:
Will this work on Windows with `/` as directory splitter? Use `path.join` instead?

Reply:
MPI does not work on Windows, so this won't work anyway.
But I'll try to write nicely the paths anyway.. (not sure __file__ works for example on Windows though. For example parcels.__file__ does not work it seems)